src/components/LikeButton.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+import { EnrichedActivity, EnrichedReaction, UR, Activity } from 'getstream';
+
+import { FeedManager } from '../Context/FeedManager';
+import { DefaultAT, DefaultUT } from '../Context/StreamApp';
+import ReactionToggleIcon from './ReactionToggleIcon';
+import { ThumbsUpIcon, Color } from './Icons';
+
+export type LikeButtonProps<
+  UT extends DefaultUT = DefaultUT,
+  AT extends DefaultAT = DefaultAT,
+  RT extends UR = UR,
+  CRT extends UR = UR
+> = {
+  activity: EnrichedActivity<AT, UT>;
+  reaction: EnrichedReaction<RT, CRT, UT>;
+} & Partial<Pick<FeedManager<UT, AT>, 'onToggleChildReaction' | 'onToggleReaction'>>;
```

Comment:
would be good to copy from old types or add some docs around the custom props https://github.com/GetStream/react-activity-feed/blob/main/src/components/LikeButton.js#L14-L26

Reply:
same thing goes for all other components